04-materials/01-exploring-extensions.md

```diff
@@ -17,8 +17,8 @@
 
 It's not just an interface for working with Notebooks.
 The Notebook interface has been around since 2011, and in 2018,
-JupyterLab was introduced to provide a more comprehensive environment, like an operating
-system, for interactive computation.
+JupyterLab was introduced as an extensible IDE (Integrated Development Environment)
```

Comment:
I'm not sure how I feel about changing from an operating system metaphor to an IDE metaphor. I think "operating system" may be a more familiar concept to most? I'm OK with this if you feel strongly, though :)

Reply:
This was my suggestion. I was going for correctness. You could call it all sorts of things like "framework" or whatever, but I thing IDE is rightest. VSCode is a "code editor" and pycharm is an "IDE" according to their docs.

Reply:
I think the operating system metaphor needs quite a bit more context and explanation, since when people think "operating system" they think things like Windows, macos, linux - which on the surface is very different than JupyterLab.

Reply:
Makes sense, thanks for your thoughts all :D